modules/homebrew.nix

```diff
@@ -536,8 +536,7 @@ in
       [](#opt-homebrew.onActivation).
 
       This module also provides a few options for modifying how Homebrew commands behave when
-      you manually invoke them, under [](#opt-homebrew.global)
-    '');
+      you manually invoke them, under [](#opt-homebrew.global)'');
```

Comment:
Is it important to remove the trailing newlines?

Reply:
Yes, otherwise the additional newline renders as an incorrect space before the end of the sentence in the documentation; you can see this [in the current online manual](https://daiderd.com/nix-darwin/manual/index.html#opt-homebrew.enable) (until the docs change). Sorry, I've been in the docs weeds for so long that it's obvious to me in a way that it definitely wasn't before :) I've pushed a more detailed commit message to explain.